docs/readme.md

```diff
@@ -4,6 +4,27 @@ title: Welcome to FAST-DNA
 sidebar_label: What is FAST-DNA
 ---
 
-FAST-DNA is a set of tooling, resources, and components used to accelerate building Web sites and applications.
+# FAST-DNA
+An unopinionated system of components, development tools, and utilities used à la carte or as a suite to build enterprise-grade websites and applications.
 
-Teams spend significant time building parts of the design system (colors, variable font, typography, grid, components, etc.) which can slow adoption of the latest design guidelines and related technologies. FAST-DNA solves this problem by providing tooling and resources to accelerate development with code that is performant and accessible. Many of the resources are written to be framework agnostic and will work with any component library and JavaScript framework (React, Angular, etc.). We also support a number of React-specific packages for UI generation.
\ No newline at end of file
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA.
+
+This principle of being unopinionated manifests in several important ways including:
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your style technology of choice, without having to re-build or duplicate the base components.
+* Framework agnostic tooling that lets you use our Development tools with any view framework.
+* The ability to replace almost any FAST-DNA package with your package of choice. Get started with our animation or layout package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
+
+### UI development and style guide tools
+When developing components and views, excellent development tooling can make all the difference. FAST-DNA offers development and style guide tools that are with FAST components, components from other frameworks or your components.
```

Comment:
"FAST-DNA offers development and style guide tools that are with FAST components" seems strangely worded - *that works with*?